### DISCUSSION TITLE
"Copy Project" and "Copy Database" Button/wizard.

### DISCUSSION THREAD

**Discussion by Aging-Developer** [supabase/supabase#29880] [Feature Requests]
As Supabase is being integrated in to more and more No-code/Low-code platforms, users experience of cloning/migrating databases is not at the same level as a developer.  It would be really helpful to a lot of users if the was a simple button/wizard in the UI to do this that doesn't require installing a local instance of Supabase.

**Comment by Kevin-McIsaac** [comment]
Agreed, see https://github.com/orgs/supabase/discussions/29879#discussioncomment-12707817

**Comment by CesarMoreno-dev** [comment]
Big thanks @alaister  and the whole Supabase team — what you’ve built is truly amazing and keeps getting better! 🙌

That said… we really need a "Copy Project" (or Duplicate Project) button — something simple and magical like Ctrl+C for entire apps! 💫

I'm about to create a new version of a web app, and it has almost the same functionality and database as the existing one. But setting everything up from the CLI feels like reinventing the wheel... with extra steps. 😅

A one-click way to clone an existing project (including the database structure and maybe auth settings) would make life so much easier — and would definitely boost productivity for both developers and non-devs.

Thanks again for all the hard work — you're making backend fun again! 🙏💚
